setup.py

```diff
@@ -1,34 +1,34 @@
 import os
-from setuptools.command.build_ext import build_ext
-from setuptools import dist, setup, Extension, find_packages
+from setuptools import setup, find_packages
 
-descr = 'Fetcher for LIBSVM datasets'
+descr = "Fetcher for LIBSVM datasets"
 
 version = None
-with open(os.path.join('libsvmdata', '__init__.py'), 'r') as fid:
+with open(os.path.join("libsvmdata", "__init__.py"), "r") as fid:
     for line in (line.strip() for line in fid):
-        if line.startswith('__version__'):
-            version = line.split('=')[1].strip().strip('\'')
+        if line.startswith("__version__"):
+            version = line.split("=")[1].strip().strip("'")
             break
 if version is None:
-    raise RuntimeError('Could not determine version')
+    raise RuntimeError("Could not determine version")
 
-DISTNAME = 'libsvmdata'
+DISTNAME = "libsvmdata"
 DESCRIPTION = descr
-MAINTAINER = 'Mathurin Massias'
```

Comment:
It may be a pain to revert but could you try to remove changes like this? We don't enforce ' instead of " so it just makes the diffs longer than it should

Reply:
👍